frontend/app/services/notifications-service.js

```diff
@@ -79,7 +79,9 @@ module.exports = function(I18n, $rootScope) {
     });
   },
   clearNotifications = function() {
-    _.remove(currentNotifications);
+    _.forEach(currentNotifications, function(notification) {
```

Comment:
If `currentNotifications is an array, you could simply use `currentNotifications.forEach`.